packages/contracts-bedrock/src/L1/OPContractsManager.sol

```diff
@@ -474,8 +477,12 @@ contract OPContractsManager is ISemver {
         return abi.encodeWithSelector(
             selector,
             _input.roles.systemConfigOwner,
-            _input.basefeeScalar,
-            _input.blobBasefeeScalar,
+            ISystemConfig.FeeScalars({
+                baseFeeScalar: _input.basefeeScalar,
+                blobBaseFeeScalar: _input.blobBasefeeScalar,
+                operatorFeeScalar: _input.operatorFeeScalar,
+                operatorFeeConstant: _input.operatorFeeConstant
+            }),
```

Comment:
I took at look at your contracts test failures. You can get them working just by making the same change to the interop version of the [OPCM here](https://github.com/yuwen01/optimism/blob/yuwen/op-fee-gpo/packages/contracts-bedrock/src/L1/OPContractsManagerInterop.sol#L48).